tests/e2e/specs/miscellaneous/RevokeOauth.spec.ts

```diff
@@ -0,0 +1,31 @@
+/** *******************************************************************
+ * copyright (c) 2020-2024 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { e2eContainer } from '../../configs/inversify.config';
+import { CLASSES } from '../../configs/inversify.types';
+import { LoginTests } from '../../tests-library/LoginTests';
+import { BASE_TEST_CONSTANTS } from '../../constants/BASE_TEST_CONSTANTS';
+import { UserPreferences } from '../../pageobjects/dashboard/UserPreferences';
+
+suite(`"Check User Preferences page" test ${BASE_TEST_CONSTANTS.TEST_ENVIRONMENT}`, function (): void {
```

Comment:
Could you please say what "${BASE_TEST_CONSTANTS.TEST_ENVIRONMENT}" is for?

Reply:
I don't know :). BASE_TEST_CONSTANTS.TEST_ENVIRONMENT is not set in constants and I don't see any using it in crw-jenkins jobs.